test_scripts/API/VehicleData/SubscribeVehicleData/004_Resumption_success_flow_2_apps_the_same_data.lua

```diff
@@ -0,0 +1,56 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/2283
+--
+-- Description: SDL sends once VI.SubscribeVehicleData request to HMI during resumption of the same vehicle data
+--  for 2 apps
```

Comment:
The line should not be longer than 121 characters, in case all text will be moved to one line the line will have 123 characters, so the comment is rejected.